website/pages/en/index.js

```diff
@@ -135,38 +135,32 @@ module.exports = class Index extends React.Component {
             <h2 className="homepage__feature-list__title">What Richie brings you</h2>
             <div className="blockElement blockElement--left twoByGridBlock">
               <div className="blockContent">
-                <h3>An LMS-agnostic education portal</h3>
+                <h3>An LMS-agnostic Education Portal</h3>
                 <p>
-                  Your course catalog can be synchronized with one or more LMS instances running
-                  different kinds of software, such as Open edX or Moodle. Richie is built to
-                  aggregate it all for your users.
+                  Course catalogs can synchronize with one or more LMS instances running different software, 
+                  such as Open edX or Moodle. Richie aggregates it all for your users.
                 </p>
-                <h3>An editor back-office</h3>
+                <h3>Author-first</h3>
                 <p>
-                  All the content in Richie is meant to be created and updated by regular editors,
-                  not software engineers. As it is based on Django CMS, you get a rich editor
-                  interface along with Richie.
+                  Content authors do not have to rely on software engineers to create and update all materials in Richie. 
+                  Instead, authors use a rich editor interface to maintain content.
                 </p>
-                <h3>Advanced access rights and moderation</h3>
+                <h3>Advanced Access Rights and Moderation</h3>
                 <p>
-                  From CMS content structured objects like organizations, courses, and categories,
-                  everything is managed through comprehensive access rights. Those can be based on
-                  individual user or tied to organizations.
+                  Everything is managed through comprehensive access rights from CMS content structured objects like 
+                  organizations, courses, and categories. Rights scale from individual users to enterprise-wide. 
                 </p>
               </div>
             </div>
             <div className="blockElement twoByGridBlock">
               <div className="blockContent">
                 <h3>A multilingual website</h3>
                 <p>
-                  Richie itself is already available in more than one language, and you can add
-                  yours by talking to us. All the content can be added and managed in as many
-                  languages as you need.
+                  Richie itself is already available in more than one language, and you can add yours by talking to us. All the content can be added and managed in as many languages as you need. Richie is available in English, French, and Spanish. Want to add yours? Reach out! Richie supports content creation and management in as many languages as you need. 
```

Comment:
We added portuguese and russian. Should we list them?

Reply:
Hm, I think that may be too many for easy reading. I would write "Richie is available in English, French, Spanish, and more." with perhaps a link to documentation on available languages.

Reply:
Perfect! You can point to : https://crowdin.com/project/richie